server/src/main/java/org/opensearch/common/blobstore/stream/read/listener/ReadContextListener.java

```diff
@@ -80,6 +86,34 @@ public void onResponse(ReadContext readContext) {
         }
     }
 
+    private ActionListener<Collection<String>> getFileCompletionListener() {
+        return ActionListener.wrap(response -> {
+            logger.trace(() -> new ParameterizedMessage("renaming temp file [{}] to [{}]", tmpFileLocation, fileLocation));
```

Comment:
No need to use ParameterizedMessage here. That's only needed when adding a trailing exception because our logger checker is not very smart.